src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -184,11 +184,6 @@ else if ( new File(arg).exists() ) {
                     "Please rename your file with the appropriate extension. If %s is NOT supposed to be a file, " +
                     "please move or rename the file at location %s", arg, arg, new File(arg).getAbsolutePath()));
         }
-        // If it's not an existing file, but looks like it was supposed to be a file (ie., it has an extension), throw an error.
-        // This works because '.' is not an allowable character in interval strings.
-        else if ( arg.indexOf('.') != -1 ) {
-            throw new UserException.CouldNotReadInputFile("The file " + arg + " does not exist");
-        }
         // Otherwise treat as an interval -> parse and add to raw interval list
```

Comment:
so this was not true i guess "'.' is not an allowable character in interval strings." Can you add a comment about it to the code  ie state that '.' is in fact allowable?